package/src/values/hooks/useComputedValue.ts

```diff
@@ -1,7 +1,7 @@
 import { useMemo } from "react";
 
 import { ValueApi } from "../api";
-import { isValue } from "../../renderer/processors";
+import { isValue } from "../../renderer/processors/Animations";
```

Comment:
Same here, is it relevant to the Mock?

Reply:
here we want to import `isValue` without pulling the React Native dependency.